deno/deepgram-topic-detection/src/mod.ts

```diff
@@ -0,0 +1,41 @@
+import { sdk } from "./deps.ts";
+
+export default async function (req: any, res: any) {
+  if (!req.variables.DEEPGRAM_API_KEY) {
+    return res.json({
+      success: false,
+      message: "The environment variables (DEEPGRAM_API_KEY) are required."
+    })
+  }
+
+  const {
+    fileUrl
+  } = JSON.parse(req.payload);
+
+  if (!fileUrl) {
+    return res.json({
+      success: false,
+      message: "The link to the audio file is missing."
+    })
+  }
+
+  try {
+    const deepgram = await sdk.default(req.variables.DEEPGRAM_API_KEY);
+    const transcription = await deepgram.transcription.preRecorded({
+      url: fileUrl
+    }, {
+      punctuate: true,
+      detect_topics: true
+    });
+
+    res.json({
+      success: true,
+      deepgramData: transcription.results
+    });
+  } catch(e: any) {
+    res.json({
+      success: false,
+      message: e
```

Comment:
It doesn't seem like `e` gets encoded properly as the error response I get looks like:

```json
{
    "response": {
        "success": false,
        "message": {}
    },
    "stdout": "",
    "stderr": ""
}
```